package.json

```diff
@@ -31,5 +31,10 @@
         "^1.0.0": "consumeStatusBar"
       }
     }
+  },
+  "apmInstallSource": {
```

Comment:
Can you remove these lines? I think you added by mistake.

Reply:
Sorry for that. I'm not sure how that happend. Fixed in branch.